service/src/test/java/bio/terra/workspace/service/job/JobServiceTest.java

```diff
@@ -64,8 +64,12 @@ void emptyStringJobIdTest() {
     assertThrows(
```

Comment:
can we add a test for nullJobId?

Reply:
I modified an existing test that was making UUIDs to skip specifying the id, so that path gets exercised.